app/localization.js

```diff
@@ -10,36 +10,34 @@
  * Localization Utilities
  */
 
-export function Localizer() {
-    // Currently configured language
-    this.language = 'en';
+export class Localizer{
+    constructor() {
+        // Currently configured language
+        this.language = 'en';
 
-    // Current dictionary of translations
-    this.dictionary = undefined;
-}
+        // Current dictionary of translations
+        this.dictionary = undefined;
+    }
 
-Localizer.prototype = {
     // Configure suitable language based on user preferences
-    setup: function (supportedLanguages) {
-        var userLanguages;
+    setup(supportedLanguages) {
 
         this.language = 'en'; // Default: US English
 
         /*
          * Navigator.languages only available in Chrome (32+) and FireFox (32+)
          * Fall back to navigator.language for other browsers
          */
-        if (typeof window.navigator.languages == 'object') {
-            userLanguages = window.navigator.languages;
-        } else {
-            userLanguages = [navigator.language || navigator.userLanguage];
-        }
 
-        for (var i = 0;i < userLanguages.length;i++) {
-            var userLang = userLanguages[i];
-            userLang = userLang.toLowerCase();
-            userLang = userLang.replace("_", "-");
-            userLang = userLang.split("-");
+        const userLanguages = (typeof window.navigator.languages == 'object')
+            ? window.navigator.languages
+            : [navigator.language || navigator.userLanguage];
+
+        for (let i = 0;i < userLanguages.length;i++) {
+            const userLang = userLanguages[i]
+                .toLowerCase()
+                .replace("_", "-")
+                .split("-");
```

Comment:
Same thing here. People can use these things as suggestions, but I don't think we should have hard enforcement.

Reply:
Same as above.